packages/publish/README.md

```diff
@@ -32,10 +32,11 @@ The functionality of mosaic-publish can also be accessed programmatically via it
 
 ```js
 import { MosaicPublisher } from '@uwdata/publish';
+import fs from 'fs';
 
 const publisher = new MosaicPublisher({
-  spec: './path/to/spec.json',
-  outputPath: './docs/public/schema'
+  spec: fs.readFileSync('./path/to/spec.yaml', 'utf8'),
```

Comment:
This seems unrelated to the current PR?

Reply:
This was meant to be on the base `publish` pr. Should be moved now.